test/mpi/cxx/spawn/spawnargvx.cxx

```diff
@@ -117,6 +117,8 @@ int main( int argc, char *argv[] )
     /* Note that the MTest_Finalize get errs only over COMM_WORLD */
     if (parentcomm == MPI::COMM_NULL) {
 	MTest_Finalize( errs );
+    } else {
+        MPI_Finalize();
```

Comment:
Is the indentation correct here?

Reply:
Yeah, the C++ tests seem to have escaped the previous whitespace cleanups. I can add another patch to fix the whitespace throughout if you think it's fine (we've done that with the test suite previously).